src/dr_client.py

```diff
@@ -519,15 +522,25 @@ def set_registered_model_global(self, registered_model_name, is_global):
                 f"Failed to find registered model by name: {registered_model_name}"
             )
 
-        if registered_model.get("isGlobal", None) == is_global:
+        payload = {}
+
+        if description is not None and description != registered_model["description"]:
```

Comment:
In my opinion, the model's definition YAML file is supposed to be the single source of true, that completely reflects the registered model in DR. It means that whenever the user removes the description field, or set it to empty, you want to erase the description in DR.

Reply:
I'm trying to make it consistent with the behavior in other places. If I e.g. remove description field from the custom model, it will not be updated.